ddpui/core/oauth/google_oauth_service.py

```
@@ -0,0 +1,244 @@
+from ddpui.models.org import Org
+from ddpui.models.org_user import OrgUser
+from ddpui.utils.custom_logger import CustomLogger
+from ddpui.utils.redis_client import RedisClient
+
+logger = CustomLogger("ddpui.core.oauth")
+
+# state: CSRF nonce bound to the orguser that started the flow (consent -> callback window)
+OAUTH_STATE_REDIS_PREFIX = "airbyte_oauth_state"
+OAUTH_STATE_TTL_SECONDS = 60  # 1 minute
+# refresh_token_ref: opaque handle to a stashed refresh_token (callback -> create-source window)
+OAUTH_REFRESH_TOKEN_REF_REDIS_PREFIX = "airbyte_oauth_refresh_token_ref"
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to: "OAUTH_REFRESH_TOKEN_REF_REDIS_PREFIX"

(S105)